plot_fits.py

```diff
@@ -291,12 +291,19 @@ def _parser():
                         'or both.')
     parser.add_argument('--ftype', help='Select which type the fits file is',
                         choices=['ARES', 'CRIRES'], default='ARES')
+    parser.add_argument('--fitsext', help='Select Crires extention',
+                        choices=['0', '1', '2', '3', '4'], default='0')
```

Comment:
Why is default 0 and not 1? Also, shouldn't this be integers or is that not possible?